server/views/pages/bookAVisit/dateAndTime.njk

```diff
@@ -19,8 +19,11 @@
     {%- set checked = true if formValues['visit-date-and-time'] and formValues['visit-date-and-time'] == slot.id else false %}
 
     {%- set tableText %}
-      {# {% if (slot.id == originalSelectedSlot.id) -%}ORIGINAL SLOT {% endif %} #}
-      {% if (doubleBooked and not checked) -%}
+      {% if (slot.id == originalSelectedSlot.id) -%}
```

Comment:
The logic in all these `if`/`elseif` getting a bit complicated but couldn't think of a better way to structure it 🤷‍♂️ . (Hoping the accompanying tests help explain!)

Reply:
Can you do `switch` in Nunjucks, that's the only other way to do it really but I've no preference. Other option is a function that takes the relevant args and returns a message.

Reply:
Thanks for the suggestions. Doesn't look like Nunjucks has `switch`. Going to leave it as is given that this date/time picker is being re-designed - an opportunity then to tidy up some of this kind of stuff.